sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/csv/CSVExprUtilsSuite.scala

```diff
@@ -58,4 +61,40 @@ class CSVExprUtilsSuite extends SparkFunSuite {
     }
     assert(exception.getMessage.contains("Delimiter cannot be empty string"))
   }
+
+  val testCases = Table(
```

Comment:
We haven't used this scalatest construct much or at all; it's fine but neutral on this vs just writing a simple loop over cases.